api/java/aggregation/ungroup.md

```diff
@@ -116,9 +109,21 @@ Result: (Note this is a JSON representation of a `List<GroupedResult>`; see the
             {"id": 11, "player": "Bob", "points": 10, "type": "free"}
         ]
     }
-}
+[
 ```
 
+__Example:__ Finding the statistical mode of an array of values:
```

Comment:
This one says "statistical mode", while the other languages say "arithmetic mode". We should make them the same.